specification/loadtestservice/LoadTestService/models.tsp

```diff
@@ -259,14 +333,28 @@ model CertificateMetadata {
   name?: string;
 }
 
-@doc("The load test configuration.")
+@added(APIVersions.v2023_04_01_preview)
+@doc("Auto stop criteria for a test.")
+model AutoStopCriteria {
+  @doc("Auto stop is enabled or not")
+  autoStopEnabled?: boolean;
+
+  @doc("Value of auto stop criteria metric")
+  errorRate?: float32;
+
+  @doc("Time window in seconds for auto stop metric")
+  errorRateTimeWindow?: int64;
+}
+
+@doc("Configurations for the load test.")
 model LoadTestConfiguration {
   @doc("""
 The number of engine instances to execute load test. Supported values are in
 range of 1-45. Required for creating a new test.
```

Comment:
engine instance count range has now been increased

Reply:
Updated to 400